src/modules/acbs/acbs-authentication.service.ts

```diff
@@ -6,8 +6,8 @@ import { AxiosResponse } from 'axios';
 import { PinoLogger } from 'nestjs-pino';
 import { catchError, lastValueFrom } from 'rxjs';
 
-import { AcbsAuthenticationFailedException } from './acbs-authentication-failed.exception';
 import { IdpConnectResponse } from './dto/idp-connect-response.dto';
+import { AcbsAuthenticationFailedException } from './exception/acbs-authentication-failed.exception';
```

Comment:
for mdm-api I've created a new folder called `auth` next to config, modules, etc folders. I wonder if ACBS authentication can go in a similar folder? it seems like a functionality that will be used a lot on TFS

Reply:
Is there a benefit to bringing it out to a top-level module instead of having it in a feature module as it is currently? I don't think the current location will prevent us re-using it throughout TFS

Reply:
As discussed in the other PRs, modules should stay in the modules folder. We leave it as it is